pkgs/development/python-modules/midea-beautiful-air/default.nix

```
@@ -0,0 +1,47 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, cryptography
+, requests
+, pytestCheckHook
+, pytest-socket
+, requests-mock
+}:
+
+buildPythonPackage rec {
+  pname = "midea-beautiful-air";
+  version = "0.10.4";
+  format = "setuptools";
```

```suggestion
  pyproject = true;
```

And put `setuptools` into `build-system` instead.